web/react/src/components/pf4/MetricsChartBase.tsx

```diff
@@ -163,37 +134,49 @@ abstract class MetricsChartBase<Props extends MetricsChartBaseProps> extends Rea
     return true;
   }
 
-  checkUnload(data: C3ChartData) {
-    const newColumns = data.columns.map(c => c[0] as string);
-    const diff = this.previousColumns.filter(col => !newColumns.includes(col));
-    if (diff.length > 0) {
-      data.unload = diff;
-    }
-    this.previousColumns = newColumns;
-  }
+  // checkUnload(data: C3ChartData) {
+  //   const newColumns = data.columns.map(c => c[0] as string);
+  //   const diff = this.previousColumns.filter(col => !newColumns.includes(col));
+  //   if (diff.length > 0) {
+  //     data.unload = diff;
+  //   }
+  //   this.previousColumns = newColumns;
+  // }
 
   render() {
     const data = this.getSeriesData();
-    this.checkUnload(data);
-    const height = 350;
-    // Note: if any direct interaction is needed with the C3 chart,
-    //  use "oninit" hook and reference "this" as the C3 chart object.
-    //  see commented code
-    // const self = this;
+    // this.checkUnload(data);
+    // const height = 350;
     return (
-      <div key={this.getControlKey()} style={{ height: '100%' }}>
+      <div key={this.getControlKey()} style={{ height: '100%', display: 'flex-inline' }}>
```

Comment:
what's the point of typestyle? It seems that the styles object from React (React.CSSProperties iirc) is already something typed, so it's saving us from adding a new lib.
Moreover as a component library, I think the less libraries we depend on, the better it is to avoid conflicts at integration.

Reply:
I see many good reasons to use `typestyle` The hashed classnames being more efficient than inline styles especially in loops and when taken as an entire application. Typestyle's composable functions are quite useful and we really should be using them more throughout the kiali. And possibly most important, is having that little separation of presentation (style) and function. The inline styles clutter up the html/javascript and make it harder to grok than is necessary.

Reply:
Also, the styles don't need to be recomputed each `render` if setup properly.

Reply:
fair enough :-) That's good for me, I've added it back